sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/junit5/DropwizardAppExtension.java

```diff
@@ -0,0 +1,127 @@
+package org.sdase.commons.server.testing.junit5;
+
+import io.dropwizard.Application;
+import io.dropwizard.Configuration;
+import io.dropwizard.cli.Command;
+import io.dropwizard.configuration.ConfigurationSourceProvider;
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.testing.DropwizardTestSupport;
+import java.util.Optional;
+import java.util.function.Function;
+import javax.annotation.Nullable;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
+ * This class only adds {@link BeforeAllCallback} and {@link AfterAllCallback} to the original
+ * {@link io.dropwizard.testing.junit5.DropwizardAppExtension}.
+ */
+public class DropwizardAppExtension<C extends Configuration>
```

Comment:
Can we add a comment linking to https://github.com/dropwizard/dropwizard/pull/3649 here and rename this to avoid the name clash with Dropwizard? That way to can run into trouble if you are using the wrong import?

Reply:
Sonar has the same opinion here 😉